File: bronze/10798/c++/main.cpp
#include <iostream>
#include <string>
int main()
{
    std::string text[5];
    int max = 0;
    for (int i = 0; i < 5; i++) {
        std::cin >> text[i];
        if (text[i].size() > max) {
            max = text[i].size();
        }
    }
    
    for (int i = 0; i < max; i++) {
        for (std::string data : text) {
            if (i >= data.size()) {
                continue;
            }
            std::cout << data[i];
        }
    }
    
    return 0;
}

File: bronze/10798/python/main.py

if (__name__ == "__main__"):
    text = []; max = 0

    for i in range(5):
        text.append(input())
        if (len(text[-1]) > max):
            max = len(text[-1])

    for i in range(max):
        for data in text:
            if (i >= len(data)): continue
            print(data[i], end='')
